content/components/component-documentation-guidelines/component-MVP-documentation-guidelines.md

```diff
@@ -0,0 +1,48 @@
+# Component documentation MVP guides
+
+## Description
+One of the strategies to help Primer documentation become self-serve is to accurately represent the breadth of patterns and components we have available through our documentation. 
+
+To do this, we are increasing our coverage to include 100% of available-for-use components temporarily in the primer/design repository. Executing on this plan requires us to be practical and concise in terms of the depth of the documentation. In order to ensure we can reach 100% coverage, we're using an MVP approach. 
+
+## Principles for this approach 
+- Accurate
+- Informative but concise
+- Visible
+
+## MVP documentation approach
+Aside from components we deem to be high touch and therefore requiring more in-depth documentation, the majority of components that are missing usage guidelines will be documented with the following minimum requirements:
+
+Description
+- brief text describing what the component is
+- link to available implementations
+- visual, if applicable 
```

Comment:
visual = anatomy? or should it be something different?

Reply:
should be updated now: https://github.com/primer/design/pull/330/commits/d14fb67179d59dd165437c3afd7235e2b5fa4c8c#diff-a52c93c589eddb08cc884196d81281bb0452cf31a10121fd80080072214962ecR19